src/current/cockroachcloud/advanced-cluster-management.md

```diff
@@ -34,7 +34,7 @@ These sections show how to scale a {{ site.data.products.advanced }} cluster hor
 You can add or remove nodes from your cluster through the Console. See [Planning your cluster]({% link cockroachcloud/plan-your-cluster.md %}) for cluster requirements and recommendations before proceeding.
 
 {{site.data.alerts.callout_info}}
-You cannot scale a multi-node cluster down to a single-node cluster. If you need to scale down to a single-node cluster, [back up]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your cluster and [restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) it into a new single-node cluster.
+You cannot scale a multi-node cluster down to a single-node cluster. For new CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024, a cluster must have at least 3 nodes per region.
```

Comment:
The second sentence here needs to be deleted. Apologies - I know we went back and forth on this, but we're not removing single node option for new orgs at this point. We are only removing the 2vcpu option for new orgs.